exporter/stackdriverexporter/stackdriver.go

```diff
@@ -126,21 +137,12 @@ func newStackdriverMetricsExporter(cfg *Config, version string) (component.Metri
 	}
 
 	userAgent := strings.ReplaceAll(cfg.UserAgent, "{{version}}", version)
```

Comment:
Bring this line into `generateClientOptions` as well?

Reply:
Done, had to change the trace exporter constructor though (which is probably for the better).